### PR TITLE
add github actions + circleci config

### PR DESCRIPTION
**Pull request by rymorale**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,80 @@
+version: 2.1
```
Comment (PravinPK) [inline .circleci/config.yml:1]: copyright info here will be nice

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+orbs:
+  android: circleci/android@2.0
```
Comment (PravinPK) [inline .circleci/config.yml:3]: can we update this to 
  `circleci/android@2.1.2`. been using this is places repo

**Review by PravinPK — APPROVED** [review]
Looks good with couple of minor things

File: .circleci/config.yml (hunk 1 above)
Comment (rymorale) [reply .circleci/config.yml:1]: will look into adding copyright info via automation script in a future pr

**Pull request merged**